lib/file.js

```diff
@@ -119,7 +119,7 @@ class File extends EventEmitter {
 
   renderTo (elem, opts, cb) {
     if (typeof window === 'undefined') throw new Error('browser-only method')
-    render.render(this, elem, opts, cb)
+    this._stream = render.render(this, elem, opts, cb)
```

Comment:
See feedback on your PR to `render-media`, but this won't work if `render` doesn't return a `Videostream` instance and instead uses one of the other strategies.